users/v1/scripts/global_variables.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+
+DESCRIPTIONS_FOLDER = Path(__file__).parent / "definitions/"
+GENERATED_FOLDER = Path(__file__).parent.parent / "generated"
+
+CORRECT_CHORD_FILENAME = "correct_chords"
+COMBOS_FILENAME = "combos"
+CYCLES_FILENAME = "cycles"
+
+# combos format is: [NAME,STR,CAN_CYCLE,IS_ACTION,KEYS...]
+NAME_POS = 0
+STR_POS = 1
+CAN_CYCLE_POS = 2
+IS_ACTION_POS = 3
+KEYS_START_POS = 4
```

Comment:
I would use an enum here, for the sake of code being more readable
```suggestion
from enum import IntEnum
class ComboType(IntEnum):
    NAME_POS = 0
    STR_POS = 1
    CAN_CYCLE_POS = 2
    IS_ACTION_POS = 3
    KEYS_START_POS = 4
```
If you dont care about the actual values, you might also import `auto` from `enum` and get automatic values by doing `NAME_POS = auto()` and so on

Reply:
I care about the value because it's the position in the CSV row. =)